HealthSDK/GiniHealthSDK/Documentation/source/Integration.md

```diff
@@ -164,105 +136,24 @@ It should be sufficient to call paymentComponentsController.loadPaymentProviderA
-    self.yourInvoicesListViewController.present(bankSelectionBottomSheet,
-                                                animated: false)
-    }
- ```
-
-## Show PaymentReviewViewController
-
-The `PaymentReviewViewController` displays an invoice's pages and extractions. It also lets users pay the invoice with the bank they selected in the `BankSelectionBottomSheet`.
-
-The `PaymentReviewViewController` presentation requires a `PaymentComponentsController` instance from the `Integrate the Payment component` step 1 and `documentId`.
-
-> **Note:** 
-> - The `PaymentReviewViewController` can be presented modally, used in a container view or pushed to a navigation view controller. Make sure to add your own navigation around the provided views.
-
-> ⚠️  **Important:**
-> - The `PaymentReviewViewController` presentation should happen in `func didTapOnBankPicker(documentId: String?)` inside
-`PaymentComponentViewProtocol` implementation without animation since SDK handles the animation during the presentation (see `Integrate the Payment component` step 3).
-
-```swift
-    func didTapOnPayInvoice(documentId: String?) {
-        guard let documentId else { return }
-        paymentComponentsController.loadPaymentReviewScreenFor(documentID: documentId, trackingDelegate: self) { [weak self] viewController, error in
-            if let error {
-                self?.showErrorsIfAny()
-            } else if let viewController {
-                viewController.modalTransitionStyle = .coverVertical
-                viewController.modalPresentationStyle = .overCurrentContext
-                self?.yourInvoicesListViewController.present(viewController, animated: false)
-            }
-        }
-    }
+         - Parameters:
+           - documentId: An optional identifier for the document associated with the payment flow.
+           - paymentInfo: An optional `PaymentInfo` object containing the payment details.
+           - navigationController: The `UINavigationController` used to present subsequent view controllers in the payment flow.
+           - trackingDelegate: The `GiniHealthTrackingDelegate` provides event information that happens on PaymentReviewScreen.
 ```
 
-> **Note:** 
-> - PaymentReviewViewController contains the following configuration options:
-> - paymentReviewStatusBarStyle: Sets the status bar style on the payment review screen. Only if `View controller-based status bar appearance` = `YES` in `Info.plist`.
-> - showPaymentReviewCloseButton: If set to true, a floating close button will be shown in the top right corner of the screen.
-Default value is false.
-
-For enabling `showPaymentReviewCloseButton`:
-
+### Optional:
+We also provide some trust markers information for creating a subview with the banks available and the numbers of them. See Figma [here](https://www.figma.com/design/fHf3b3XxE59wymH7gvoMrJ/iOS-Gini-Health-SDK-5.0-UI-Customisation?node-id=12906-13711&node-type=instance&t=fLL9Yl3dPpmV51U0-0)
```

Comment:
[suggestion]: What do you say about this rewrite: "We also provide trust marker information for creating a subview that displays the available banks and their respective numbers."